src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -82,6 +82,7 @@ public static String getResource(String key) {
             {"R_loginFailed", "Login failed"}, {"R_exitedMoreSeconds", "Exited in more than {0} seconds."},
             {"R_exitedLessSeconds", "Exited in less than {0} seconds."},
             {"R_invalidArgumentExecutor", "The argument executor is not valid"},
+            {"R_invalidArgumentCatalog", "The argument catalog is not valid"},
```

Comment:
There is no need for this anymore. You should be able to use SQLServerResource directly by calling `formatErrorMsg()`. See PR #1034.

Reply:
This error message doesn't exist in SQLServerResource, and is thrown by SQL Server, so this should be fine.

Reply:
Are these for string comparisons? There are a few issues where SQL Server would return error messages with an extra period depending on the version (mainly DW). E.g "The connection is closed" vs "The connection is closed.".

Reply:
Yep - I'm comparing with "contains"

Reply:
https://github.com/Microsoft/mssql-jdbc/blob/51d6c40a1193c879d9b10eca254c510c84fe5a15/src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java#L159